pyproject.toml

```
@@ -13,8 +14,11 @@ matplotlib = "^3.9.2"
 tqdm = "^4.67.0"
 pyyaml = "^6.0.2"
 mdanalysis = "^2.7.0"
-scikit-learn = "^1.5.2"
+scikit-learn = ">=1.5.2,<1.5.3"
```

isn't this just 1.5.2?
```suggestion
scikit-learn = "1.5.2"
```